apps/api/src/applicant/dto/ienapplicant-create.dto.ts

```diff
@@ -84,8 +91,12 @@ export class IENApplicantCreateUpdateAPIDTO extends IENApplicantCreateUpdateDTO
     ],
   })
   @IsArray()
-  @IsOptional()
-  nursing_educations?: JSON;
+  @ArrayMinSize(1, { message: 'At least 1 Education is required' })
+  @ArrayNotEmpty({ message: 'Education is required' })
+  @ValidateIf(e => e.nursing_educations.length > 0)
+  @ValidateNested()
+  @Type(() => NursingEducationDTO)
```

Comment:
Swagger doc doesn't show the schema for nested nursing education dto.
We might need to define separate dto in api project. It means sharing concept of common project doesn't work for validators and swagger api.